trusted/service/sys/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "mc-sgx-service-sys"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+mc-sgx-service-sys-types = { path = "types" }
```

Comment:
I see this pattern repeating, but don't understand it. Why not fold the `*-sys-types` crates into `*-sys`? They both use `bindgen`

Reply:
My understanding is that some of the types may get re-used in other crates while the functions/linking won't.

Perhaps @jcape can explain it better

Reply:
How about we default to merging `sys-types` into `sys`, until we need to separate them?

Reply:
On further thought, keeping `sys-types` separate SGTM

Reply:
sounds good thanks!